utilities/src/main/java/io/syndesis/qe/bdd/validation/OpsValidationSteps.java

```diff
@@ -0,0 +1,213 @@
+            sa.assertThat(optPanel).isPresent();
+            optPanel.ifPresent(node -> {
+                sa.assertThat(node.get("targets").findValues("expr").stream().map(JsonNode::asText)).contains(row.get(2).replace("\\\"", "\""));
+                sa.assertThat(node.get("type").asText()).isEqualTo(row.get(3));
+            });
+        }
+        sa.assertAll();
+    }
+
+    private void verifyAlertIsRaised(String name) {
+        JsonNode res = queryEndpoint.getAlerts();
+        assertThat(res.get("status").asText()).isEqualToIgnoringCase("success");
+        JsonNode alerts = res.get("data").get("alerts");
+        assertThat(Streams.stream(alerts.elements())).anyMatch(alert ->
+                                                                   name.equals(alert.get("labels").get("alertname").asText()) &&
+                                                                       alert.get("state").asText().matches("pending|firing"));
+    }
+
+    private static void scaleDown(String name) {
+        log.info("Scaling down {}", name);
+        OpenShiftUtils.getInstance().deploymentConfigs().inNamespace(TestConfiguration.openShiftNamespace()).withName(name).scale(0);
+        OpenShiftWaitUtils.waitUntilPodIsDeleted(name);
+    }
+
+    private boolean wasAlertRaised(String alertName) {
+        return Streams.stream(queryEndpoint.getAlerts().get("data").get("alerts").elements())
+            .map(alert -> alert.get("labels").get("alertname").asText()).distinct().filter(alertName::equals).count() > 0;
+    }
+
+    @Then("verify monitoring alerts are working correctly")
+    public void verifyAlerts() throws TimeoutException, InterruptedException {
+        scaleDown("syndesis-operator");
+        scaleDown("syndesis-db");
+        OpenShiftWaitUtils.waitFor(() -> wasAlertRaised("FuseOnlineDatabaseInstanceDown"), 10 * 1000 * 60);
+        verifyAlertIsRaised("FuseOnlineDatabaseInstanceDown");
+        verifyAlertIsRaised("FuseOnlinePostgresExporterDown");
+        log.info("Database tests passed\n Executing REST requests and waiting for endpoint alerts");
+        for (int i = 0; i < 1000; i++) {
+            if (wasAlertRaised("FuseOnlineRestApiHighEndpointErrorRate")) {
+                break;
+            }
+            EndpointClient.getClient()
+                .property("disable-logging", true)
+                .target(Constants.LOCAL_REST_URL).path("api").path("v1").path("connections")
+                .request()
+                .header("X-Forwarded-User", "pista")
+                .header("X-Forwarded-Access-Token", "kral")
+                .header("SYNDESIS-XSRF-TOKEN", "awesome")
+                .get();
+        }
+        verifyAlertIsRaised("FuseOnlineRestApiHighEndpointErrorRate");
+        OpenShiftUtils.getInstance().deploymentConfigs().inNamespace(TestConfiguration.openShiftNamespace()).withName("syndesis-operator").scale(1);
```

Comment:
What if the test failed on some exception before this scaling of the operator and db? Wouldn't the pods be missing for the next tests? I would add there some finalizer in order to be sure that pods are scaling back in any cases.